### PR TITLE
Traefik alerts

### PR DESCRIPTION
**Pull request by miransar**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/traefik/HighHttp400.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: High 400 Error Rate
+
+description: |+
+  This alert is triggered when Traefik has high HTTP 4xx error rate service.
```
Comment (thezackm) [inline alert-policies/traefik/HighHttp400.yml:4]: please update for readability: 

```
This alert is triggered when a Traefik service has an HTTP 4xx error rate above 5% for over 5 minutes. This threshold should be adjusted to your own requirements.
```

File: alert-policies/traefik/HighHttp400.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+nrql:
+  query: "FROM Metric 
+   SELECT filter(
+     sum(
+       traefik_service_requests_total), 
+     WHERE code Not RLIKE '4.*') * 100 
+   / filter(
+     sum(
+       traefik_service_requests_total), WHERE code RLIKE '.*') 
+   WHERE instrumentation.name = 'remote-write'"
```
Comment (thezackm) [inline alert-policies/traefik/HighHttp400.yml:15]: this needs an additional `FACET` to help users isolate the alerts

```
... FACET instance, service
```

File: alert-policies/traefik/HighHttp500.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: High 500 Error Rate
+
+description: |+
+  This alert is triggered when Traefik has high HTTP 5xx error rate service.
```
Comment (thezackm) [inline alert-policies/traefik/HighHttp500.yml:4]: please update for readability:

```
This alert is triggered when a Traefik service has an HTTP 5xx error rate above 5% for over 5 minutes. This threshold should be adjusted to your own requirements.
```

File: alert-policies/traefik/HighHttp500.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+nrql:
+  query: "FROM Metric 
+   SELECT filter(
+     sum(
+       traefik_service_requests_total), 
+     WHERE code Not RLIKE '5.*') * 100 
+   / filter(
+     sum(
+       traefik_service_requests_total), WHERE code RLIKE '.*') 
+   WHERE instrumentation.name = 'remote-write'"
```
Comment (thezackm) [inline alert-policies/traefik/HighHttp500.yml:15]: this needs an additional `FACET` to help users isolate the alerts

```
... FACET instance, service
```

File: alert-policies/traefik/TooManyRequests.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+name: Too Many Open requests
+
+description: |+
+  This alert is triggered when Traefik has too many open requests.
```
Comment (thezackm) [inline alert-policies/traefik/TooManyRequests.yml:4]: please update for readability:

```
This alert is triggered when a Traefik service has more than 50 open requests for over 1 minute. This threshold should be adjusted to your own requirements.
```

File: alert-policies/traefik/TooManyRequests.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+type: STATIC
+nrql:
+  query: "SELECT 
+  if((average(traefik_service_open_connections) > 50), 
+  average(traefik_service_open_connections), 1) 
+  FROM Metric 
+  WHERE instrumentation.name = 'remote-write'"
```
Comment (thezackm) [inline alert-policies/traefik/TooManyRequests.yml:11]: this needs an additional `FACET` to help users isolate the alerts

```
... FACET instance, service
```

**Review by thezackm — CHANGES_REQUESTED** [review]
@miransar - some small edits requested on the alerts for the `prometheus-growth` project; otherwise this looks good.

File: alert-policies/traefik/HighHttp500.yml (hunk 4 above)
Comment (miransar) [reply alert-policies/traefik/HighHttp500.yml]: Thanks, resolved.

File: alert-policies/traefik/HighHttp400.yml (hunk 1 above)
Comment (miransar) [reply alert-policies/traefik/HighHttp400.yml]: Thanks, resolved.

File: alert-policies/traefik/HighHttp400.yml (hunk 2 above)
Comment (miransar) [reply alert-policies/traefik/HighHttp400.yml]: Thanks, resolved.

File: alert-policies/traefik/HighHttp500.yml (hunk 3 above)
Comment (miransar) [reply alert-policies/traefik/HighHttp500.yml]: Thanks, resolved.

File: alert-policies/traefik/TooManyRequests.yml (hunk 5 above)
Comment (miransar) [reply alert-policies/traefik/TooManyRequests.yml]: Thanks, resolved.

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**